spaces/domains.json

```
@@ -663,6 +663,7 @@
   "vote.bsc-mvb.io": "mvb.eth",
   "vote.dios.finance": "diosfinance.eth",
   "vote.mysticalorbs.xyz": "mysticcollective.eth",
+  "vote.foundance.org": "foundance.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.foundance.org": "foundance.eth",
```

---

Oh my word 🤦 , apologies!